btree/btree.c

```diff
@@ -5580,10 +5590,14 @@ int64_t btree_create_tree_tick(struct m0_sm_op *smop)
 
 		m0_rwlock_write_lock(&bop->bo_arbor->t_desc->t_lock);
 		bop->bo_arbor->t_desc->t_height = bop->bo_arbor->t_height;
+		_slot.s_node                    = oi->i_nop.no_node;
+		_slot.s_idx                     = 0;
+		node_capture(&_slot, bop->bo_tx);
```

Comment:
If node_capture() is added here, then we need to add node_capture() in btree_create_tree_tick() as node_header is modified there also. The addition is to be done after line 5633.